pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.black]
+line-length = 99
```

Comment:
OMG! I am on the 79 team.. but I think I can try to accommodate 99 now that I have a widescreen monitor.